<launch>
    <rosparam command="load" file="$(find sample_overtaking_node)/settings.yaml"/>
    <node pkg="sample_overtaking_node" name="main_car_overtaking" type="sample_overtaking_node">
      <param name="car_name" type="str" value="f1tenth_car" />
      <param name="nav_drive_topic" type="str" value="/vesc/high_level/ackermann_cmd_mux/input/nav_0" />
      <param name="scan_topic" type="str" value="/scan" />
      <param name="speed" type="double" value="2.5" />
      <param name="wait_time" type="int" value="2000" />
    </node>
</launch>
